eng/pipelines/linux.yml

```diff
@@ -89,6 +98,7 @@ jobs:
         - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
           - linuxDefaultQueues: Centos.7.Amd64+RedHat.7.Amd64+Debian.8.Amd64+Debian.9.Amd64+Ubuntu.1604.Amd64+Ubuntu.1804.Amd64+Ubuntu.1810.Amd64+OpenSuse.42.Amd64+SLES.12.Amd64+SLES.15.Amd64+Fedora.27.Amd64+Fedora.28.Amd64
           - linuxArm64Queues: Ubuntu.1604.Arm64
+          - linuxArmQueues: '\(Debian.9.Arm32\)Ubuntu.1604.Arm32.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm32v7-b049512-20190321153542'
```

Comment:
`20190321153542` seems pretty specific. How often will we need to update this value?

Reply:
That is just the automatic GUID generated by the repo that builds the images. It shouldn't be that often, should just be updated if we fixed something in the image and we want to consume the newer.

Reply:
This changes every time when we need to make changes to images - like add new dependencies or make some global changes. So far, we have two versions for arm32 @danmosemsft and less than 10ish for x64. Note that for other queue similar version tag lives in .azure-ci.yml